stingray/lightcurve.py

```diff
@@ -236,7 +236,7 @@ def __getitem__(self, index):
                              "object !")
 
     @staticmethod
-    def make_lightcurve(toa, dt, tseg=None, tstart=None):
+    def make_lightcurve(toa, dt, tseg=None, tstart=None, tend=None):
```

Comment:
@dhuppenkothen @OrkoHunter I have made a few changes here (to accommodate the option for `tend`). It was necessary for one of my tests to pass. Please review if it's okay.